src/energies.jl

```diff
@@ -22,7 +22,7 @@ Base.pairs(energies::Energies) = pairs(energies.energies)
 Base.iterate(energies::Energies) = iterate(energies.energies)
 Base.iterate(energies::Energies, state) = iterate(energies.energies, state)
 Base.haskey(energies::Energies, key) = haskey(energies.energies, key)
-
+Base.sum(energies::Energies) = sum(values(energies))
```

Comment:
We had this before and I removed it. I really don't like this. For me energies is a Dict and a Dict iterates over pairs and not over the thing it maps to, so sum(energies) is not a simple value. I don't mind if you define a function `total` for this purpose.

Reply:
The fact that the pattern `sum(values(energies))` appears so much is a sign we should do something about it. I really don't like adding a generic `total` function: it makes little sense for DFTK to export such a generic word. And if you use another package that does this it conflicts. See, this is where we need OOP :-p We could maybe emulate this by overloading getproperty. Eg we could do energies.total, energies.Kinetic, etc. and there'd be nice tab completion at the REPL.